lib/src/base-styles/_nx-tile.scss

```diff
@@ -36,7 +36,7 @@
 }
 
 .nx-tile-header {
-  align-items: center;
+  align-items: top;
```

Comment:
This isn't a valid value for this property

Reply:
> This isn't a valid value for this property

Ops, I fixed it here: 1922addd2f72058347203b17a89c352515e3e9aa